src/guidance/index.md.njk

```diff
@@ -0,0 +1,10 @@
+---
+title: Guidance
+description: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
can we write a description to replace this placeholder text.

e.g. guides to help design, build and operate services.

i'm not sure what a good description of this collection of guidance is.  I'd be happy to just have the fake latin replaced by something in English, and we can review for the best description later.

Reply:
Replaced with: "Guidance to help you with your design process in HM Land Registry." in line with the guidance index page content, pending Mette's review.